src/control-flow-basics/loops.md

```diff
@@ -25,39 +25,8 @@ fn main() {
 
 ## `for`
```

Comment:
I think these headings and links can be omitted -- students and instructors typically page through the course with the next and previous links, so the next link will take them to `for`, and on to `loop`.